components/contribution-flow/ContributionFlowButtons.js

```diff
@@ -40,14 +41,18 @@ class ContributionFlowButtons extends React.Component {
       case STEPS.PROFILE:
         return <FormattedMessage id="ContributionFlow.YourInfo" defaultMessage="Your info" />;
       case STEPS.PAYMENT:
-        return <FormattedMessage id="ContributionFlow.Payment" defaultMessage="Payment" />;
+        return this.props.isCrypto ? (
+          <FormattedMessage id="ContributionFlow.Checkout" defaultMessage="Checkout" />
+        ) : (
+          <FormattedMessage id="ContributionFlow.Payment" defaultMessage="Payment" />
+        );
```

Comment:
I don't see what justify different words here. I guess it was in the design but still.

Reply:
I am fine with changing this, but I feel the rationale behind this design. My thinking is that the `Payment` header gives the user the idea that they need to choose the payment whereas the checkout page is instantaneous checking out. No payment, it's already checked out. I don't know, it's a slight difference, but the working makes sense to me. They don't do a payment there, they just checkout. Payment is actually done in their wallet side. 😉

Reply:
What does "Checkout" means actually?

https://dictionary.cambridge.org/dictionary/english/checkout

> the area of a website where you order and pay for goods, or the process of paying for goods

If we take the first part of the definition, without a shopping cart, when you enter the contribution flow, you're already in the "checkout" process. Being the whole process, it's difficult to justify it as a name for a sub-step.  

Looking at the second part of the definition, it's the same as "Payment".

While I'm willing to understand that the payment process with crypto is a bit different than usual, after looking at definitions, I don't understand how introducing this particular new word in the button is helping our users here.

If we want to reuse what's already there:

- "Payment" as in Step Profile
- "Contribute 20 BTC" as in Step Payment method

Reply:
@znarf : Makes sense, I've made the changes. Thanks much for the explanation. 😉

Reply:
@SudharakaP Thank you for doing the change. I feel a bit sorry to have engaged for such a small change.

But let's keep this in mind has a good guideline, we should avoid introducing new words and new copy if there is not a clear value for the end user. 

This is not final, if we have a good case for a different copy, and help users better understand what they're doing, I'm up for it.

Reply:
Sure no worries. I'll keep this in mind for the future. 👍🏽 😃